src/main/kotlin/com/careem/gradle/dependencies/treeTldr.kt

```diff
@@ -115,7 +85,7 @@ private fun partitionDifferences(
       removedVersion != null -> {
         mutableRemovedMap.remove(artifact)
         upgrades.add(
-          VersionedDependency(artifact, "$version, (changed from $removedVersion)"))
```

Comment:
changing it to store the `otherVersion` in field of `VersionedDependency` to decouple printing logic from business logic